### ISSUE TITLE
Consider changing the default port

### ISSUE THREAD

**Issue by FliegendeWurst** [TriliumNext/Trilium#5298]
### Description

Using a port >32k is not recommended, as this range is used for [ephemeral ports](https://en.wikipedia.org/wiki/Ephemeral_port).
This can lead to random issue like https://github.com/zadam/trilium/issues/821 or https://github.com/zadam/trilium/issues/3651.

### TriliumNext Version

any

### What operating system are you using?

Ubuntu

### What is your setup?

Local (no sync)

### Operating System Version

any

**Comment by contributor** [comment]
> changing the "default" port (which seems to make sense for the reason outlined above) would apparently require changing the port in web-clipper as well

yep. And changing default port in web clipper is good reason to do long awaiting web clipper release https://github.com/zadam/trilium-web-clipper/issues/63
